### PR TITLE
add a restart icon to all test related views

### PR DESCRIPTION
**Pull request by coolo**
this is a followup of #307:

- untabify the views
- add more restart buttons
- remove the inconsistent back button
- simplify the restart javascript by providing the right url from the controller

### CODE REVIEW

**Comment by coolo** [comment]
oh, I also added a "Clone of" entry - to make it possibly to return where you came from

File: lib/OpenQA/Controller/API/V1/Job.pm [hunk 1]
```diff
@@ -206,7 +206,7 @@ sub restart {
     }
 
     my @res = OpenQA::Scheduler::job_restart($target);
-    $self->render(json => {result => \@res});
+    $self->render(json => {result => $res[0], test_url => $self->url_for('test', testid => $res[0]) });
```
Comment (aaannz) [inline lib/OpenQA/Controller/API/V1/Job.pm]: This does not handle the situation when more jobs are restarted at once, see :203

Comment (coolo) [inline lib/OpenQA/Controller/API/V1/Job.pm]: yeah, but the javascript never did. So keep having arrays?

Comment (aaannz) [inline lib/OpenQA/Controller/API/V1/Job.pm]: I don't know if we use array version anywhere, but to be on safe side I would keep arrays. Or if you don't want to, then remove the every_param part.

**Comment by coolo** [comment]
ok, seems to work: f83545f

**Comment by coolo** [comment]
damn, I missed running

**Pull request merged**